EventKit/eventkit-cloud
# Add auth check to provider rerun

Validates oauth token when rerunning individual providers. Disables 'rerun file(s)' button in ellipsis menu when the user does not have permissions. 

**To test:**
* setup geoaxis locally. Message me for the required configuration. Then open two tabs. Run an export in tab one. Wait for it to finish. After the job has finished logout in the tab two. In tab one rerun an individual provider. Verify that a NotAuthenticated error is thrown and that you are not able to re-run the provider. 

* Create a datapack, share it with another user but don't give that user admin privileges. Log out, log back in with the second user. Navigate to the datapack. Verify that if you click the ellipsis menu on individual providers the 'rerun file(s)' option is disabled + does nothing when you click it. Go back to the original user. Give admin privileges to the second user and repeat, verify that now the second user is able to rerun individual providers. 

BLOCKED: 
Need to discuss how to handle logout + modify UI to accomplish that effect.
